src/java_1/dataTypes/DataTypes.java

```diff
@@ -13,38 +13,58 @@
 //		long halfByte = 0101;
```

Comment:
These variables should've been refactored as per the instructions on line 17 (before edits).